doc/pages/keys.asciidoc

```diff
@@ -213,40 +207,40 @@ the Shift modifier and moving will extend the selection instead.
 == Changes
 
 *i*::
-    enter insert mode before current selection
+    enter insert mode before selections
 
 *a*::
-    enter insert mode after current selection
+    enter insert mode after selections
 
 *d*::
-    yank and delete current selection
+    yank and delete selections
 
 *c*::
-    yank and delete current selection and enter insert mode
+    yank and delete selections and enter insert mode
 
 *.*::
     repeat last insert mode change (*i*, *a*, or *c*, including the
     inserted text)
 
 *<a-d>*::
-    delete current selection (not yanking)
+    delete selections (not yanking)
 
 *<a-c>*::
-    delete current selection and enter insert mode (not yanking)
+    delete selections and enter insert mode (not yanking)
 
 *I*::
-    enter insert mode at current selection begin line start
+    enter insert mode at selections begin line start
```

Comment:
I think this should be "enter insert mode at the beginning of the lines containing the start of each selection"